### PR TITLE
features : Customer and Repository class created

### PR DESCRIPTION
**Pull request by sagarxt**
#5

### CODE REVIEW

File: backend/src/main/java/com/moshop/backend/model/Customer.java
Comment (Abinashkjcjanscinanconanc) [inline backend/src/main/java/com/moshop/backend/model/Customer.java:1]: Remove space from line 27

**Review by Abinashkjcjanscinanconanc — COMMENTED** [review]
Change according to comment

File: backend/src/main/java/com/moshop/backend/model/Customer.java
Comment (sagarxt) [reply backend/src/main/java/com/moshop/backend/model/Customer.java:1]: line 27 space removed

**Review by Abinashkjcjanscinanconanc — APPROVED** [review]
LGTM

**Pull request merged**